tools/wheels/upload_wheels.sh

```diff
@@ -41,20 +43,20 @@ set_upload_vars() {
     fi
 }
 upload_wheels() {
-    echo ${PWD}
-    if [[ ${ANACONDA_UPLOAD} == true ]]; then
-        if [ -z ${TOKEN} ]; then
+    echo "${PWD}"
+    if [[ $ANACONDA_UPLOAD == true ]]; then
+        if [[ -z TOKEN ]]; then
```

Comment:
The `-z` primary still does expect a string, not a variable name, so it should be given the string `$TOKEN` expanded as a variable.